turtlebot3: #namespace

    task_and_robot_environment_name: 'TurtleBot3World-v0'
    # ros_ws_abspath: "/home/gz/simulation_ws"
    ros_ws_abspath: "$HOME/simulation_ws"
    running_step: 0.04 # amount of time the control will be executed
    pos_step: 0.016     # increment in position for each command

    #dqn parameters
    gamma: 0.999        # discount rate default 0.999
    epsilon_start: 0.99  # exploration rate default 0.9
    epsilon_end: 0.05   # default 0.05
    epsilon_decay: 50   # default 200
    n_episodes: 2000      # default 30
    target_update: 30   # default 10
    batch_size: 512     # default 128

    running_step: 0.06 # Time for each step default 0.06
    learning_rate: 0.00025 # default 0.01

